linera-client/src/client_context.rs

```diff
@@ -442,7 +442,7 @@ where
         blob_path: PathBuf,
     ) -> Result<CryptoHash, Error> {
         info!("Loading data blob file");
-        let blob_content = BlobContent::load_from_file(&blob_path)
+        let blob_content = BlobContent::load_data_from_file(&blob_path)
```

Comment:
It's because we hash the `BlobContent` below... but I think I can just create the `BlobContent` at the end and just pass the bytes around. Will simplify the code a bit also